changelog.d/12868.misc

```diff
@@ -0,0 +1,2 @@
+Enable the `batch_send` endpoint on synapse worker processes. Contributed by Nick @ Beeper.
+
```

Comment:
Can you remove the trailing newline please? Annoyingly GH is refusing to let me make a suggested change to do that :(